code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -88,7 +85,8 @@
 	UnregisterSignal(former_ethereal, COMSIG_ATOM_EMAG_ACT)
 	UnregisterSignal(former_ethereal, COMSIG_ATOM_EMP_ACT)
 	UnregisterSignal(former_ethereal, COMSIG_LIGHT_EATER_ACT)
-	QDEL_NULL(ethereal_light)
+	if(ethereal_light)
+		QDEL_NULL(ethereal_light)
```

Comment:
unsure why you do this, if you knew already then it's fine and you can ignore me but qdel(null) is valid and fine, don't need to guard against it.
we'll use this pattern in very hot deconstructors because the proc call has overhead, but it's not required

Reply:
why'd you resolve this?

Reply:
uhhh you said this pattern is ok, even if not requried.

Reply:
i'll get rid of it then

Reply:
yeah I just wanted to make sure you knew it wasn't is all